msvc-full-features/Cataclysm-vcpkg-static.vcxproj

```diff
@@ -156,13 +157,20 @@
     <EmbedManifest>false</EmbedManifest>
     <TargetName>cataclysm-tiles</TargetName>

```

Comment:
Probably could change the name of this in notiles also, but don't have energy to spin it back up right now.